queries.yaml

```diff
@@ -292,6 +295,15 @@ layers:
       - vectordatasource.transform.truncate_min_zoom_to_2dp
     sort: vectordatasource.sort.transit
     area-inclusion-threshold: 1
+  admin_areas:
+    geometry_types: [Polygon, MultiPolygon]
+    simplify_before_intersect: false
+    simplify_start: 9
```

Comment:
If we start including these features at zoom 5 we should probably also simplify starting at zoom 5? Or if we're not simplifying at all before intersect and we're dropping this layer does it matter?

Reply:
I think it would be better to not simplify at all. I've removed this parameter in #1501, and made sure that it drops the `admin_areas` layer _before_ simplification happens. Just to make doubly-sure, I've set `simplify_start` beyond any zoom we'll ever render!